src/libraries/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -25,6 +25,7 @@
     <Compile Include="System\IO\Ports\StopBits.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' != 'true' and '$(TargetsWindows)' == 'true'">
+    <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\NullableAttributes.cs" Link="System\Diagnostics\CodeAnalysis\NullableAttributes.cs" />
```

Comment:
NIT I thought Eirik was solving this globally

Reply:
The change he made only applies for projects with nullable enabled.

Reply:
Got it, thank you

Reply:
Actually you can just add `<Nullable>annotations</Nullable>` to the project and it will automatically define the constant and include the compile item for you. That was support for projects like this that we haven't annotated yet but a common source which is annotated needs this attributes.

https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.targets#L182

Theorically you could set `Nullable` to anything, but `annotations` was the value/pattern we defined when he did his change.